build.gradle

```diff
@@ -243,6 +243,27 @@ bootJar {
     archiveName = "${baseName}.${extension}"
 }
 
+private static void copyFromUrl(GString localScriptPath, String scriptUrl) {
```

Comment:
if i run it locally, should i use gradle to get scripts? probably we can try to make it in java code?